internal/getproviders/package_authentication.go

```diff
@@ -37,16 +37,29 @@ type PackageAuthenticationResult struct {
 
 func (t *PackageAuthenticationResult) String() string {
 	if t == nil {
-		return "unauthenticated"
+		return "unsigned"
```

Comment:
This change is incorrect. A nil result does not mean merely unsigned, it means entirely unauthenticated—not even a checksum verification.